src/test/java/org/jitsi/meet/test/pageobjects/web/ToolbarV2.java

```diff
@@ -0,0 +1,416 @@
+/*
+ * Copyright @ 2018 Atlassian Pty Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.meet.test.pageobjects.web;
+
+import org.jitsi.meet.test.util.*;
+import org.jitsi.meet.test.web.*;
+import org.openqa.selenium.*;
+
+import java.util.*;
+
+/**
+ * Represents the toolbar in a particular {@link WebParticipant}.
+ *
+ * @author Leonard Kim
+ */
+public class ToolbarV2 extends Toolbar {
```

Comment:
I inherited out of laziness to inherit the type. The old toolbar should be removed soon-ish. I'm eyeballing a month from now unless someone has other plans. I'm not sure if this is an acceptable short term hack.